### DISCUSSION TITLE
trying to run java -jar command in execInContainer method

### DISCUSSION THREAD

**Discussion by calypso0000** [testcontainers/testcontainers-java#6233] [Q&A]
Hello everyone,

I am having some troubles with running the java -jar command inside of the container. 
I have test.jar under /opt but couldnt run it.
My code line is:

             container.execInContainer("java", "-jar", "/opt/test.jar", ">>", "/opt/log.txt");

It didnt work. Any recommendation?

Thank you

**Comment by pioorg** [comment]
Have you tried running `var result = container.execInContainer("java", "-jar", "/opt/test.jar");`
and then checking the output of the command using `result.getStdout()`?
